BranchInfo.props

```diff
@@ -7,12 +7,12 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
 
     <PreReleaseLabel Condition="'$(PackageVersionStamp)' != ''">$(PackageVersionStamp)</PreReleaseLabel>
-    <PreReleaseLabel Condition="'$(PreReleaseLabel)' == ''">preview1</PreReleaseLabel>
+    <PreReleaseLabel Condition="'$(PreReleaseLabel)' == ''">alphautf8string</PreReleaseLabel>
     <IncludePreReleaseLabelInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true' or '$(PackageVersionStamp)' != ''">true</IncludePreReleaseLabelInPackageVersion>
     <IncludeBuildNumberInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true'">true</IncludeBuildNumberInPackageVersion>
 
     <ReleaseSuffix>$(PreReleaseLabel)</ReleaseSuffix>
-    <ReleaseBrandSuffix>Preview 1</ReleaseBrandSuffix>
+    <ReleaseBrandSuffix>Alpha Feature Utf8String</ReleaseBrandSuffix>
     <Channel>master</Channel>
     <BranchName>master</BranchName>
```

Comment:
@joshfree - You are going to want to update both the `Channel` and `BranchName` as well.

Reply:
@eerhardt what do those feed into? Is that for the azure blob storage location? If so I'm not sure we need to adjust them for this short-lived dev branch as there shouldn't be conflicts.

Reply:
@eerhardt is Channel the dotnet/versions channel or something different?

Reply:
`$(Channel)` is used here:

https://github.com/dotnet/core-setup/blob/09c681b7a46163f9ef4e2fc41ca4ad3cba346734/publish/publish.proj#L146-L158

It is used to decide which "channel" to update the `latest` build for, which has links on the README.  For example:

https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-runtime-latest-win-x64.exe

If you don't update this, builds coming from this branch will show up as the latest `master` build on that link, which is definitely not what we want.

______________

`$(BranchName)` -  I thought this used to be used to update the `dotnet/versions` repo but we do that by using `$(SourceBranch)` in our build definitions:

https://github.com/dotnet/core-setup/search?utf8=%E2%9C%93&q=VersionsRepoPath&type=

I can't find any usages of `$(BranchName)` anymore.

Reply:
I wonder if we shouldn't use SourceBranch for the Channel by default as well and eliminate the need for changing these across branches.